frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.scss

```diff
@@ -0,0 +1,12 @@
+.co-guided-tour-drawer {
+  &__panel-content {
+    overflow-y: hidden !important;
```

Comment:
yes. but i figured it would also cause the footer of the child component to be scrollable, so put the no scroll to test with a form and it slipped in. I'll remove it. I'll test it after creating the guided tour component.